integration/tests/bft/gossipsub/signature/requirement/orchestrator.go

```diff
@@ -21,10 +21,10 @@ const (
 	// The numOfAuthorizedEvents allows us to wait for a certain number of authorized messages to be received, this should
 	// give the network enough time to process the unauthorized messages. This ensures us that the unauthorized messages
 	// were indeed dropped and not unprocessed.
-	numOfAuthorizedEvents = 50
```

Comment:
It would be helpful to add warnings / cautions of keeping `numOfAuthorizedEvents` and `numOfUnauthorizedEvents` not too high. Were they reduced to decrease test times? Address flakiness?

Reply:
https://github.com/onflow/flow-go/pull/4652/commits/53a2434294c4240a01e813a8d23c0eaabdc17421